users/serializers.py

```diff
@@ -267,6 +277,28 @@ def get_unused_coupons(self, instance):
             return fetch_and_serialize_unused_coupons(instance)
         return []
 
+    def validate(self, data):
+        """Validate an existing user"""
+        name = data.get("name")
+        if name:
+            try:
+                openedx_validation_msg = validate_name_with_edx(name)
+                openedx_validation_msg = OPENEDX_NAME_VALIDATION_MSGS_MAP.get(
+                    openedx_validation_msg, openedx_validation_msg
```

Comment:
We can propagate the error message saying there was an error with the full name, We might as well read the error message from the edX response and show it to the user.

Reply:
Yes, this is what that functionality does. It will get actual error message `Enter a valid name` mapping from `OPENEDX_NAME_VALIDATION_MSGS_MAP`, if it is different openedx error message then this will show to user.